cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -2727,6 +2627,11 @@ CacheAllocator<CacheTrait>::evictNormalItemForSlabRelease(Item& item) {
   auto token = evictToNvmCache ? nvmCache_->createPutToken(item.getKey())
                                : typename NvmCacheT::PutToken{};
 
+  if (skipIfTokenInvalid && evictToNvmCache && !token.isValid()) {
```

Comment:
I think we can actually remove this logic. I don't remember clearly the rationale behind checking token validity right after creating it in `advanceIteratorAndTryEvictRegularItem()`.

Maybe it was to avoid evicting item pre-maturely (when we cannot insert it into nvm-cache). But this should be fairly rare in practice.

Reply:
Done (in separate commit). Should I also remove the corresponding stat `evictFailConcurrentFill` or leave it for compatibility? 

Also this stat has the following comment next to it: "Eviction failures because this item has a potential concurrent fill from nvm cache. For consistency reason, we cannot evict it. Refer to NvmCache.h for more details."